build.ps1

```diff
@@ -281,21 +291,26 @@ function Nupkg
     if(-not (Test-Path $nuget)) {
         Die "Please install nuget. More information available at: http://docs.nuget.org/docs/start-here/installing-nuget"
     }
+    
+    $NupkgVersion = $Version
+    if($Suffix) {
+        $NupkgVersion = $Version + '-' + $Suffix
```

Comment:
Effectively you are creating a `pre-release` version by adding a `-` (See https://semver.org/#spec-item-9 if your not familiar with `SemVer`).

This is already supported by the build script, just pass in `$Version` that contains the appended suffix.